### DISCUSSION TITLE
Installaltion failing

### DISCUSSION THREAD

**Discussion by walliesdevelopments** [AzuraCast/AzuraCast#7071] [Q&A]
i am getting following errors when trying it install azuracast, how can i fix this

root@SAU-F4C3A-OR:/var/azuracast# ./docker.sh install
Checking installation requirements for AzuraCast...
[PASS] Operating System: Linux
[PASS] Architecture: x86_64
[PASS] Command Present: curl
[PASS] Command Present: awk
[PASS] User Permissions
[PASS] Installation Directory
[PASS] All requirements met!
Docker is already installed! Continuing...
Docker Compose v2 is already installed. Continuing...
Your current release channel is 'Rolling Release'. Switch to 'Stable' release channel? [y/N] N
[+] Running 0/0
 ⠋ installer Pulling                                                                                               0.0s
error getting credentials - err: exit status 1, out: `Cannot autolaunch D-Bus without X11 $DISPLAY`
[+] Running 0/1
 ⠿ Network azuracast_installer_default  Error                                                                      3.9s
failed to create network azuracast_installer_default: Error response from daemon: Failed to Setup IP tables: Unable to enable NAT rule:  (iptables failed: iptables --wait -t nat -I POSTROUTING -s 172.24.0.0/16 ! -o br-e53dd6a72773 -j MASQUERADE: iptables: No chain/target/match by that name.
 (exit status 1))
[+] Running 0/0
 ⠋ updater Pulling                                                                                                 0.0s
 ⠋ web Pulling                                                                                                     0.0s
error getting credentials - err: exit status 1, out: `Cannot autolaunch D-Bus without X11 $DISPLAY`
[+] Running 0/1
 ⠿ Network azuracast_default  Error                                                                                7.1s
failed to create network azuracast_default: Error response from daemon: Failed to Setup IP tables: Unable to enable NAT rule:  (iptables failed: iptables --wait -t nat -I POSTROUTING -s 172.25.0.0/16 ! -o br-e50b15b198a1 -j MASQUERADE: iptables: No chain/target/match by that name.
 (exit status 1))
[+] Running 0/0
 ⠋ web Pulling                                                                                                     0.0s
 ⠋ updater Pulling                                                                                                 0.0s
error getting credentials - err: exit status 1, out: `Cannot autolaunch D-Bus without X11 $DISPLAY`

**Comment by BusterNeece** [comment]
This is definitely an upstream issue with Docker; I'd recommend searching for the error message here (`Cannot autolaunch D-Bus without X11 $DISPLAY`) and seeing if there are any solutions on the Docker side. It's likely nothing we can do on the AzuraCast end.
